packages/ttd-cli.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,37 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolName = "ttd"
+    $category = "Debuggers"
+    $toolVersion = "1-11-319-0"
+
+    # From https://aka.ms/ttd/download
+    $zipUrl = "https://windbg.download.prss.microsoft.com/dbazure/prod/$toolVersion/TTD.msixbundle"
+    $zipSha256 = "f7b80731c3a6994b3763c4100073b101965327d6556fa4bfb553d70ce49be366"
+
+    # Create a temp directory to download zip
+    $tempDownloadDir = Join-Path ${Env:TEMP} "$($toolName)_$($toolVersion)"
+
+    # Download and unzip
+    $packageArgs = @{
+        packageName    = ${Env:ChocolateyPackageName}
+        unzipLocation  = $tempDownloadDir
+        url            = $zipUrl
+        checksum       = $zipSha256
+        checksumType   = 'sha256'
+    }
+    Install-ChocolateyZipPackage @packageArgs | Out-Null
+    VM-Assert-Path $tempDownloadDir
+
+    # Get the unzipped x64 msix (also there are "TTD-ARM64.msix" and "TTD-x86.msix")
+    $unzippedInstaller = Join-Path $tempDownloadDir "TTD-x64.msix" -Resolve
```

Comment:
You don't need to unzip and select the version manually, you can use `Add-AppxPackage` with the `msixbundle` directly. The code of this package should look very similar to https://github.com/mandiant/VM-Packages/pull/1058 which we haven't merged because of an issue in the CI. This also shows that the introduction of a helper function for these kind of packages would be useful.

Reply:
Done.